content/apps/using-openid.md

```diff
@@ -41,12 +40,12 @@ You can consult the list of [available authentication scopes](/apps/access-scope
 
 ## Authenticate after the first Authorization request
 
-If an user is trying to access your App from his Akeneo PIM, and you want to authenticate him, start an
+If a user is trying to access your App from his Akeneo PIM and wants to authenticate him, start an
```

Comment:
```suggestion
If a user is trying to access your App from his Akeneo PIM and wants to authenticate, start an
```
Though the meaning isn't exactly the same as the original version (before grammarly)

Reply:
Exactly.
For context, when I started fixing some typos in this page, Romain tried to use Grammarly to automatically fix other typos but It changed the meaning of some sentences.

I then separated the changes into 2 PRs:
- easy typos: https://github.com/akeneo/pim-api-docs/pull/547/files (merged)
- controversial changes suggested by Grammarly (current PR)

Reply:
I will close this PR, since I don't know what to do with it.